<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="wh-chunqingz-01" name="quark.jautor.testcases.数据库测试9668b465dff04a3083698d3d0f6fd9ac" tests="1" failures="0" timestamp="24 二月 2018 09:28:32 GMT" time="25.145" errors="1">
  <testcase name="sIT01数据库测试50cc3da50f884764906ac8593452870a" time="25.145" classname="quark.jautor.testcases.数据库测试9668b465dff04a3083698d3d0f6fd9ac">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at quark.jautor.database.QueryDB.exec(QueryDB.java:110)
	at quark.jautor.core.BaseExecuter.execAll(BaseExecuter.java:146)
	at quark.jautor.testcases.数据库测试9668b465dff04a3083698d3d0f6fd9ac.sIT01数据库测试50cc3da50f884764906ac8593452870a(数据库测试9668b465dff04a3083698d3d0f6fd9ac.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </error>
  </testcase> <!-- sIT01数据库测试50cc3da50f884764906ac8593452870a -->
</testsuite> <!-- quark.jautor.testcases.数据库测试9668b465dff04a3083698d3d0f6fd9ac -->
